src/cascadia/TerminalControl/Resources/en-US/Resources.resw

```diff
@@ -138,9 +138,13 @@
     <comment>The placeholder text in the search box control.</comment>
   </data>
   <data name="DragFileCaption" xml:space="preserve">
-    <value>Copy path to file</value>
+    <value>Paste path to file</value>

```

Comment:
Maybe "Insert path to file" and "Insert text"? What do you think

Reply:
discussed offline, "insert" means "put the text here and scoot the existing text to the right", so I'm leaving it as is